orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/WaitForManifestStableTask.java

```diff
@@ -87,16 +88,23 @@ public TaskResult execute(@Nonnull Stage stage) {
           allStable = false;
           messages.add(identifier + ": " + status.getStable().getMessage());
         }
+
+        if (status.getFailed() != null && status.getFailed().isState()) {
```

Comment:
I don't think the other one can NPE, if `status.getStable() == null`, the `||` will short circuit and we assume the manifest isn't stable since nothing is reported.